docs/environment-variables.rst

```diff
@@ -0,0 +1,42 @@
+Environment Variables
+=====================
+
+It may happen that your documentation depends on an authenticated service to be built properly.
```

Comment:
This introductory paragraph is weird. On one hand, it encourages users to use this functionality for secrets, while at the end of the guide it's shown how to reach such secrets. Wouldn't this allow an attacker to send a malicious pull request and read such secrets?

On the other hand, even if I misunderstood something and this is not a concern, I would put the "Read the Docs provides a way to define..." first, then this as an example.

Reply:
I just read this and I agree that we should warn the user about this on the point 

> #. Check the :guilabel:`Public` option if you want to expose this environment variable
>   to :doc:`builds from pull requests </pull-requests>`.

and modify the first paragraph as well.

Reply:
Changed :+1: